aws/resource_aws_ecs_service.go

```diff
@@ -274,6 +274,17 @@ func resourceAwsEcsService() *schema.Resource {
 				},
 			},
 
+			"propagate_tags": {
```

Comment:
Two changes:
* Since it doesn't appear this attribute supports updates, we should add `ForceNew: true`
* We can add plan time validation for this attribute via:

```go
ValidateFunc: validation.StringInSlice([]string{
	ecs.PropagateTagsService,
	ecs.PropagateTagsTaskDefinition,
}, false),
```

I forget if it'll complain about `NONE`, if it does, just add `"NONE"` to the list. 👍